docs/volume/volume-snapshots.md

```diff
@@ -8,33 +8,39 @@ keywords:
 Description: Take a snapshot for a volume from the Volume page.
 ---
 
-## Create Volume Snapshots
+A volume snapshot represents a snapshot of a volume on a storage system. After creating a volume, you can create a volume snapshot and restore a volume to the snapshot's state. With volume snapshots, you can easily copy or restore a volume's configuration.
 
-:::note
+## Create volume snapshots
 
-A recurring snapshot is currently not supported and is tracked via [harvester/harvester#572](https://github.com/harvester/harvester/issues/572)
+You can create a volume snapshot from an existing volume by following these steps:
 
-:::
+1. Go to the **Volumes** page.
+1. Choose the volume that you want to take a snapshot of and select **⋮ > Take Snapshot**.
+
+ ![create-volume-snapshot-1](/img/v1.2/volume/create-volume-snapshot-1.png)
+1. Enter a **Name** for the snapshot.
 
-After creating a volume, you can create volume snapshots by following the steps below:
+ ![create-volume-snapshot-2](/img/v1.2/volume/create-volume-snapshot-2.png)
+1. Select  **Create** to finish creating a new volume snapshot.
 
-1. Click the `⋮` button and select the `Take Snapshot` option.
-1. Configure the `Name` of the new image and click `Create`.
+1. You can check the status of this operation and view all volume snapshots from the **Backup & Snapshot > Volume Snapshots** page. When the **State** becomes **Active**, the volume snapshot is ready to use.
```

Comment:
You can check the status of this operation and view all volume snapshots belong to the volume from the volume's **Snapshots** tab. When the **Ready To Use** becomes **√**, the volume snapshot is ready to use.
![image](https://github.com/harvester/docs/assets/15064560/b03812d9-fe55-464b-944a-1183d40bc4e7)

---
Currently, the Active field cannot be used to determine whether a volumesnapshot is ready

Reply:
@LucasSaintarbor

Reply:
Thanks @futuretea! Updated per [0a31124](https://github.com/harvester/docs/pull/302/commits/0a311248fd3533febdb8e17529850e91102e62e6)